code/modules/vehicles/scooter.dm

```diff
@@ -0,0 +1,87 @@
+/obj/vehicle/scooter
+	name = "scooter"
+	desc = "A popular child's toy back on the planets, handcrafted by some greyshirt."
+	icon_state = "scooter"
+	vehicle_move_delay = 2.2 //slightly slower than other vehicles, due to being made with pipes, rods, and metal
+	var/pipetype = 1 //so the pipe created on deconstruction doesn't have a blank icon_state, changes on construction of the scooter
+	var/pipename	 //same
+/obj/vehicle/scooter/attackby(obj/item/I, mob/user, params)
+	if(istype(I, /obj/item/weapon/wrench))
+		user << "<span class='notice'>You begin to remove the pipe..</span>"
+		if(do_after(user, 40/I.toolspeed, target = src))
+			new/obj/item/scooter_frame(get_turf(src))
+			var/obj/item/pipe/Q = new/obj/item/pipe
+			Q.pipe_type = pipetype
+			Q.pipename = pipename
+			user.put_in_hands(Q)
+			new/obj/item/stack/sheet/metal(get_turf(src),2)
+			user << "<span class='warning'>It all falls apart!</span>"
+			qdel(src)
+
+/obj/vehicle/scooter/handle_vehicle_layer()
+	if(dir == SOUTH)
+		layer = MOB_LAYER+0.1
+	else
+		layer = OBJ_LAYER
+
+/obj/vehicle/scooter/handle_vehicle_offsets()
+	..()
+	if(buckled_mob)
+		switch(buckled_mob.dir)
+			if(NORTH)
+				buckled_mob.pixel_x = 0
+				buckled_mob.pixel_y = 4
+			if(EAST)
+				buckled_mob.pixel_x = -2
+				buckled_mob.pixel_y = 4
+			if(WEST)
+				buckled_mob.pixel_x = 2
+				buckled_mob.pixel_y = 4
+			if(SOUTH)
+				buckled_mob.pixel_x = 0
+				buckled_mob.pixel_y = 4
+//CONSTRUCTION
+#define SCOOTER_STATE_FRAME		1
+#define SCOOTER_STATE_WHEELS	2
+/obj/item/scooter_frame
+	name = "scooter frame"
+	icon = 'icons/obj/vehicles.dmi'
+	icon_state = "scooter_frame_1"
+	var/construction_state = SCOOTER_STATE_FRAME
+
+/obj/item/scooter_frame/attackby(obj/item/I, mob/user, params)
+	if(construction_state == SCOOTER_STATE_FRAME)
```

Comment:
no, just make the second if an Else If.
two conditions is too little to employ a switch.